plugins/destination/postgresql/client/client.go

```diff
@@ -132,6 +135,19 @@ func (c *Client) currentDatabase(ctx context.Context) (string, error) {
 	return db, nil
 }
 
+func (c *Client) currentSchema(ctx context.Context) (string, error) {
+	var resetVal string
+	err := c.conn.QueryRow(ctx, "select reset_val from pg_settings where name='search_path'").Scan(&resetVal)
+	if err != nil {
+		return "", err
+	}
+	schemaTokens := strings.Split(resetVal, ",")
```

Comment:
This won't always work. Better to just do `select current_schema();`
https://stackoverflow.com/a/69841555/7506

Reply:
Good catch.